egs2/TEMPLATE/asr1/db.sh

```diff
@@ -117,6 +117,35 @@ WENETSPEECH=
 SPEECHCOMMANDS=downloads
 TOTONAC=downloads
 
+# For only CMU environment
+if [[ "$(hostname)" == tir.lti.cs.cmu.edu ]]; then
```

Comment:
Thanks, I think this is a good example to show that how other institutes customize it by following it!